### DISCUSSION TITLE
Collection where each entry is a link without a title?

### DISCUSSION THREAD

**Discussion by nocksock** [statamic/cms#10738] [Q&A] [answered]
Let's say, I want a collection `Feed Entries` to provide full control over the entries in an RSS feed. Most entries will be just a reference/link to another page. However, title is a required field, which makes this less straightforward than I anticipated. While I could copy and paste the title easily, I was wondering if there was a way to:

a: automatically fill that field with the value of the reference/link
b: make the title optional

Or if there's an entirely different approach to my use case.

**Comment by duncanmcclean** [comment, marked as answer]
You can hide the Title field in the blueprint (set Visibility to "Hidden"), then you can give the collection a title format to use the link instead: https://statamic.dev/collections#titles

**Reply by nocksock** [reply]
That's perfect! Thanks.
